balboa-http/src/main/scala/com/socrata/balboa/server/MetricsServlet.scala

```diff
@@ -85,10 +76,6 @@ class MetricsServlet extends JacksonJsonServlet
       contentType = response.contentType
       response.result
     }
-    finally
-    {
-      periodMeter.update(System.currentTimeMillis() - begin, TimeUnit.MILLISECONDS)
```

Comment:
This looks like a timer metric to me. You can update it to the style described here:

http://scalatra.org/2.4/guides/monitoring/metrics.html#toc_214

Same with the other two I think.